tests/integration/tier1/destructive/host-metering/test_run_conversion_with_metering.py

```diff
@@ -39,6 +40,18 @@ def setup_test_metering_endpoint():
         )
 
 
+def set_the_option_in_config():
+    """
+    Set the configure_host_metering option through the config file.
+    """
+    config_path = "/etc/convert2rhel.ini"
+    config = configparser.ConfigParser()
+    config.read(config_path)
+    config.set("host_metering", "configure_host_metering", "force")
```

Comment:
Cannot we make this as a generic helper function?, I can see that this can be used in other tests. Maybe make it as a class, that can sets different values?

Reply:
I have that in my backlog :sweat_smile: 
When we move away from the envars completely, and we'll need to set overrides through the config, I'll refactor this to a proper parameterizable helper function.
At the moment given the time constraints I see this as sufficient.

Reply:
Ok, this can be fixed later on then.